pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -275,65 +333,44 @@ func TestGetTaskFunc_Bundle(t *testing.T) {
 
 	testcases := []struct {
 		name         string
-		localTasks   []runtime.Object
-		remoteTasks  []runtime.Object
+		taskYAML     string
 		ref          *v1beta1.TaskRef
 		expected     runtime.Object
-		expectedKind v1beta1.TaskKind
+		expectedKind v1.TaskKind
 	}{
 		{
-			name:       "remote-task-bundle",
-			localTasks: []runtime.Object{simpleNamespacedTask},
-			remoteTasks: []runtime.Object{
-				&v1beta1.Task{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: "simple",
-					},
-					TypeMeta: metav1.TypeMeta{
-						APIVersion: "tekton.dev/v1beta1",
-						Kind:       "Task",
-					},
-				},
-				&v1beta1.Task{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: "dummy",
-					},
-					TypeMeta: metav1.TypeMeta{
-						APIVersion: "tekton.dev/v1beta1",
-						Kind:       "Task",
-					},
-				},
-			},
+			name: "remote-task-bundle",
```

Comment:
Isn't this functionality covered by TestGetTaskFunc_RemoteResolution? I think this test class can be removed.

Reply:
sorry I think i might have misunderstood from the discussion at https://github.com/tektoncd/pipeline/pull/6778#discussion_r1220221678 and https://github.com/tektoncd/pipeline/pull/6778#discussion_r1220369285. I thought this would be keeping the v1beta1 bundle syntax which is deprecated in v1? 🤔

Reply:
The comment you linked to says "The new bundle syntax test class can be removed when support for the bundle syntax is removed in the v1 PR", which is what I'm suggesting here. It looks like what this PR is currently doing is adding redundant tests for the bundles resolver, which I don't think is necessary. We do still want to make sure bundles syntax works, but this is tested in the bundles integration tests.